GCSViews/ConfigurationView/ConfigPlanner.cs

```diff
@@ -890,7 +890,7 @@ private void chk_ADSB_CheckedChanged(object sender, EventArgs e)
                 var server = "127.0.0.1";
                 if (Settings.Instance["adsbserver"] != null)
                     server = Settings.Instance["adsbserver"];
-                if (DialogResult.Cancel == InputBox.Show("Server", "Server IP?", ref server))
+                if (DialogResult.Cancel == InputBox.Show("Server", "Server IP? (Empty if using built-in ADSB API)", ref server))
                     return;
                 Settings.Instance["adsbserver"] = server;
 

```

Comment:
As an additional improvement, if the user leaves the IP blank, we should skip the prompt for the port. Someone will inevitably blank the port out too, which triggers an exception on the int.Parse on `MainV2.cs:385`. Best to throw an `if(server != "")` or something like that around the next prompt.